package.json

```diff
@@ -25,14 +25,20 @@
   "prettier": {
     "singleQuote": true
   },
+  "dependencies": {},
   "devDependencies": {
     "@release-it-plugins/lerna-changelog": "^5.0.0",
     "@release-it-plugins/workspaces": "^3.2.0",
-    "release-it": "^15.5.0"
+    "release-it": "^15.5.0",
+    "release-plan": "^0.8.0"
   },
   "engines": {
     "node": "16.* || 18.* || >= 20"
   },
+  "volta": {
+    "node": "16.20.2",
+    "yarn": "1.22.17"
```

Comment:
this seems wrong since we switched to pnpm

Reply:
indeed! removed